src/SiteExtensions/LoggingAggregate/src/Microsoft.AspNetCore.AzureAppServices.SiteExtension/Microsoft.AspNetCore.AzureAppServices.SiteExtension.csproj

```diff
@@ -15,6 +15,7 @@
     <GeneratePackageOnBuild>true</GeneratePackageOnBuild>
     <DisablePackageReferenceRestrictions>true</DisablePackageReferenceRestrictions>
     <NoSemVer20>true</NoSemVer20>
+    <IsShippingPackage>true</IsShippingPackage>
```

Comment:
This means the package is now in the Shipping folder and will be released to our customers. CTI testing can use the package from its previous NonShipping location and it's part of the infrastructure deployed to Azure for each release (not intended for use elsewhere). So, why are we making this late change?

Reply:
We are deploying this from the NonShipping location? From my discussion with @shirhatti it seems like we need to ship this to NuGet as well which is why I made this change, so that it will be automatically pushed to NuGet upon release.